dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
Two possibilities: one is having this ambiguity and making sure the model exposes the actual state in an accessor function; the other is to avoid the ambiguity by not having additional enums.  I'm honestly not sure which is better, but having multiple ways to get to the same state does seem a bit weird.